ui/widgets/controlgroup.js

```diff
@@ -61,6 +61,9 @@ return $.widget( "ui.controlgroup", {
 		this._callChildMethod( "destroy" );
 		this.childWidgets.removeData( "ui-controlgroup-data" );
 		this.element.removeAttr( "role" );
+		if ( this.options.items.controlgroupLabel ) {
+			this.element.find( ".ui-controlgroup-label-contents" ).contents().unwrap();
```

Comment:
i wonder if we should be more explicit and do `this.element.find( this.options.items.controlgroupLabel ).find( ".ui-controlgroup-label-contents" ).contents().unwrap();`